docs/admin/install.rst

```diff
@@ -650,6 +650,27 @@ After enabling it, you should set :setting:`ENABLE_HTTPS` in the settings:
    :setting:`ENABLE_HTTPS`,
    :ref:`production-site`
 
+
+Set properly SECURE_HSTS_SECONDS
+++++++++++++++++++++++++++++++++
+
+If your site is served over SSL, you have to consider setting a value for :setting:`SECURE_HSTS_SECONDS`
+in the settings.py and enabling HTTP Strict Transport Security.
+By default its set to 0 as shown below.
+
+.. code-block:: python
+
+   SECURE_HSTS_SECONDS = 0
+
+If set to a non-zero integer value, the `weblate.middleware.SecurityMiddleware`
```

Comment:
It's Django middleware taking care of this, not Weblate one.

Reply:
Cool. I have added the Django(Security Middleware) link for it.